src/hooks/api/getters/useApplication/useApplication.ts

```diff
@@ -0,0 +1,50 @@
+import useSWR, { mutate, SWRConfiguration } from 'swr';
+import { useState, useEffect } from 'react';
+import { formatApiPath } from '../../../../utils/format-path';
+import handleErrorResponses from '../httpErrorResponseHandler';
+
+const useApplication = (name: string, options: SWRConfiguration = {}) => {
+    const fetcher = async () => {
+        const path = formatApiPath(`api/admin/metrics/applications/${name}`);
```

Comment:
You can move this out of the fetcher so that we don't recreate it on every fetch.